src/main/xsl/preprocess/topicpullImpl.xsl

```diff
@@ -77,8 +83,8 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
            match="*[contains(@class, ' topic/table ')][*[contains(@class, ' topic/title ')]]"
            use="'include'"/>
   
-  <xsl:key name="id" match="*[@id]" use="@id"/>
-  <xsl:key name="topic" match="*[@id][contains(@class, ' topic/topic ')]" use="@id"/>
+  <xsl:key name="elementsById" match="*[@id]" use="@id"/>
```

Comment:
This key will also match topics. Either the name should be the old `id` or then the match pattern should be changed to ignore topics.